migtests/tests/import-file/run-import-file-test

```diff
@@ -94,6 +94,37 @@ main() {
 	import_data_file --data-dir ${TEST_DIR} --format csv --delimiter ',' \
 			--file-table-map "t1_escape_backslash.csv:t1_escape_backslash" --file-opts "escape_char=\\"
 
+	# Test for csv file with backspace as quote character and escape character in text field
```

Comment:
`in text field` ? Is it required.

Reply:
oh ok, it was giving an impression to me of file format (CSV and text). See if rewording is possible.